src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-druid-integration.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Apache Druid integration
+tags:
+  - Druid integration
+  - apache druid integration
+  - New Relic integrations
+metaDescription: Use New Relic infrastructure agent to get a dashboard with druid metrics.
+freshnessValidatedDate: 2024-04-21
+---
+
+import infrastructureDruidDashboard from 'images/infrastructure_screenshot-full_apache-druid-dashboard.webp'
+
+Integrating Apache Druid with New Relic enhances your ability to monitor, analyze, and optimize the performance of your Druid clusters. By leveraging New Relic's powerful monitoring and alerting features, you can ensure the reliability and efficiency of your Druid-based analytics platform.
+
+<img
+    title="Apache Druid dashboard"
+    alt="Apache Druid dashboard"
+    src={infrastructureDruidDashboard}
+/>
+
+## Set up the Apache Druid integration
+
+Complete the following steps to set up the Apache Druid integration:
+
+<Steps>
+    <Step>
+## Install the infrastructure agent
+To use the Apache Druid integration, you need to also [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with Apache Druid-specific data.    
```

Comment:
```suggestion
To use the Apache Druid integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the Apache Druid integration extends your monitoring with data specific to your Druid clusters.    
```

Reply:
this nested clause is a little bit of a mouthful: integration you'll install in the next step 

am offering some alt wording for readability, but it's just an option. :)